IntroToCSharp/Pages/Activities/Variables.razor

```diff
@@ -0,0 +1,148 @@
+@inherits CaptainCoder.Page
+@page "/activities/variables"
+
+<MudText Typo="Typo.h4" Style="font-weight:bold">Variables</MudText>
+<Section Title="Introduction">
+    <MudText>An essential part of a computer program is being about to store data or “remember” it for future use. Each
+        computer has the capacity to “remember” a lot of things. Computers can remember numbers, letters, whole words,
+        and a lot more. However, they can't remember things that the programmer doesn’t explicitly tell them to. So,
+        when we
+        are coding we must use variables to label and store bits and pieces of essential data that we will need later.
+    </MudText>
```

Comment:
This isn't entirely true.

A computer can remember things without a variable; files are a good example of this. However, we use variables to keep a "label" that allows us to look up that data.

I like to use the term "managing complexity" frequently when I teach about programming. Variables are a way to manage complexity.

How do you feel about this edit:

```
An essential part of programming is being able to store data for future use. For example, in a video game you might track the amount of experience a player has gained throughout their adventure. To do this, programmers utilize the computers Random Access Memory, often called RAM. Throughout this book we will simply refer to this as "memory".

When programming, we need to be able to store, update, and read parts of the computer memory to keep track of information. To help manage this complexity, we take use variables to give a label to those parts of memory.
```

Reply:
There is another thing that might be worth mentioning: Program memory is volatile. That is, when you start a program, the computer has no "memories". Throughout the program, it gains memories and when the program exits, it completely forgets everything.